action.yaml

```diff
@@ -70,7 +70,7 @@ runs:
           preview="preview.png"
           echo "PREVIEW_NAME=$preview" >> "$GITHUB_ENV"
           echo "PREVIEW_URL=https://$org.github.io/$repo$extra/$preview?v=$preview_date" >> "$GITHUB_ENV"
-          if [ "${{ env.BACKLOG_STATUS }}" -eq 0 ]; then
+          if [ "${{ env.BACKLOG_STATUS }}" == 0 ]; then
```

Comment:
hm, the shell would only complain if `BACKLOG_STATUS` is not a number.
I wonder when that happens. But it seems to work as expected, so...

Reply:
`""` is not a number, so this is a fix when the variable is not defined (which unfortunately happens).